.github/workflows/test_and_deploy.yml

```diff
@@ -85,45 +85,18 @@ jobs:
       run:
         shell: bash -el {0}
     steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-          fetch-tags: true
-          ref: ${{ github.ref }}
-
-      - name: Setup micromamba
-        uses: mamba-org/setup-micromamba@v2
-        with:
-          environment-name: test
-          init-shell: bash
-          create-args: >-
-            python=3.10
-
       - name: Download conda package artifact
         uses: actions/download-artifact@v4
         with:
           name: artifact-conda-package
           path: /tmp/local-channel/linux-64
 
-      - name: Install the package
-        run: |
-          micromamba install --yes -c conda-forge conda-build conda-index
-          python -m conda_index /tmp/local-channel
-          micromamba install --yes -c /tmp/local-channel -c mantid-ornl -c conda-forge ${{ env.PKG_NAME }}
-
-      - name: Verify the installation
-        run: |
-          conda_version=$(micromamba list "${{ env.PKG_NAME }}" | awk -v pkg="${{ env.PKG_NAME }}" '$1 == pkg { print $2 }')
-          echo "Conda version: $conda_version"
-          python_version=$(python -c "import ${{ env.PKG_NAME }}; print(${{ env.PKG_NAME }}.__version__)")
-          echo "Python version: $python_version"
-          if [ "$conda_version" != "$python_version" ]; then
-            echo "Version mismatch!"
-            exit 1
-          else
-            echo "Versions match."
-          fi
+      - name: Verify conda package
+        uses: neutrons/conda-verify@main
+        with:
+          local-channel: /tmp/local-channel
+          package-name: ${{ env.PKG_NAME }}
+          extra-channels: mantid-ornl
```

Comment:
we need extra entry , perhaps `run:` that allows one to run additional tests. The default tests of making sure one can import the package and making sure about the version congruence is not enough. I also want to test I can import some dependencies.